src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -8,286 +8,384 @@
     <ul class="list add-person mt-4">
       <li class="add-person-container">
         <section class="meta-container">
-
+          <!-- Add/edit person -->
           <label class="add-person-header" v-if="isPerson">
-            <span v-if="isNaN(activeIndex)">Add Person</span>
+            <span v-if="isNew">Add Person</span>
             <span v-else>Edit Person</span>
           </label>
 
+          <!-- Add/edit org -->
           <label class="add-org-header" v-if="isOrg">
-            <span v-if="isNaN(activeIndex)">Add {{ orgTypesLabel }}</span>
+            <span v-if="isNew">Add {{ orgTypesLabel }}</span>
             <span v-else>Edit {{ orgTypesLabel }}</span>
           </label>
 
-          <div class="meta-container__inner">
-            <v-form
-              id="org-person-form"
-              ref="orgPersonForm"
-              v-model="orgPersonFormValid"
-              v-on:submit.prevent
-            >
-              <!-- Person name -->
+          <!-- Form container -->
+          <div class="d-flex">
+            <v-form id="org-person-form" ref="orgPersonForm" v-model="orgPersonFormValid" v-on:submit.prevent>
```

Comment:
Also, all major blocks are now articles (instead of just divs), which gives the HTML some semantics (useful for accessibility) by related content.

Ref: https://www.w3schools.com/html/html5_semantic_elements.asp